src/generics/middleware/authenticator.js

```diff
@@ -341,8 +343,8 @@ module.exports = async function (req, res, next, token = '') {
 					!orgDetails.success ||
```

Comment:
@borkarsaish65 optimize this check 
if (
    !orgDetails?.success ||
    !orgDetails?.data ||
    Object.keys(orgDetails.data).length === 0 || // Check if data is an empty object
    !orgDetails.data.related_orgs?.length || // Check if related_orgs array is empty or non-existent
    !orgDetails.data.organizations?.length    // Check if organizations array is empty or non-existent
) {

Reply:
done